docs/source/sparse.rst

```diff
@@ -167,6 +171,160 @@ receiving a particular layout. We are working on an API to control the result la
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.],
+	    [0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1., 0., 0., 1., 1.]],
+	   device='cuda:0', dtype=torch.float16)
+    >>> a = to_sparse_semi_structured(a)
+    SparseSemiStructuredTensor(shape=torch.Size([16, 16])
+     metadata=tensor([[-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370],
+	    [-4370]], device='cuda:0', dtype=torch.int16)
+     values=tensor([[1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.],
+	    [1., 1., 1., 1., 1., 1., 1., 1.]], device='cuda:0',
+	   dtype=torch.float16))
+
+Semi-Structured Sparse Operations
+-------------------------------------------
+
+We currently support the following ops:
+
+- torch.addmm(bias, dense, sparse)
+- torch.addmm(bias, sparse, dense)
+- torch.mm(dense, sparse)
+- torch.mm(sparse, dense)
+- aten.linear.default(dense, sparse, bias)
+- aten.t.default(sparse)
+- aten.t.detach(sparse)
+
+To use these ops, simply pass the output of ``to_sparse_semi_structured(tensor)``  instead of using ``tensor`` once your tensor has 0s in a semi-structured sparse format, like this:
+
+    >>> a = torch.Tensor([0, 0, 1, 1]).tile((64, 16)).half().cuda()
+    >>> b = torch.rand(64, 64).half().cuda()
+    >>> dense = torch.mm(a, b)
+    >>> a = to_sparse_semi_structured(a)
+    >>> sparse = torch.mm(a, b)
```

Comment:
nit: This might make the user think that the result of torch.mm here is also sparse.

You could also write
```
>>> c = torch.mm(a, b)
>>> b_sparse = to_sparse_semi_structured(b)
>>> torch.allclose(c, torch.mm(a, b_sparse))
```